android/filament-android/src/main/java/com/google/android/filament/LightManager.java

```diff
@@ -326,6 +326,18 @@ public void destroy(@Entity int entity) {
          */
         @IntRange(from = 1)
         public int vsmMsaaSamples = 1;
+
+        /**
+         * Standard deviation of the VSM blur. Zero do disable.
+         * The maximum value is 21, which corresponds to a gaussian blur filter width
+         * of 125 pixels. The relation between the filter width and the standard deviation
+         * is roughly: stddev = (kernelWidth + 1) / 6.
+         * Some common values for blurStandardDeviation:
+         *  3x3 gaussian : 0.6667
```

Comment:
Should this API take a blur radius and we compute the standard deviation from that? It would be more user friendly.

Reply:
Yes, I can probably do that.